src/applications/mhv-medical-records/manifest.json

```diff
@@ -0,0 +1,7 @@
+{
+  "appName": "MHV Medical Records isolated app test",
+  "entryFile": "./app-entry.jsx",
+  "entryName": "medical-records-test",
+  "rootUrl": "/my-health/medical-records-test",
+  "productId": "c054e1e8-7be2-11ed-a1eb-0242ac120002-test"
```

Comment:
This might be okay but all other apps use a UUID. Can we generate one for this?

Reply:
done!